src/ui/src/js/controllers/job/export_systems.js

```diff
@@ -0,0 +1,45 @@
+
+jobExportSystemsController.$inject = [ '$scope', '$rootScope', 'JobService' ];
+
+/**
+ * jobExportSystemsController - Controller for the job export page.
+ * @param  {Object} $scope            Angular's $scope object.
+ * @param  {Object} $rootScope        Angular's $rootScope object.
+ * @param  {Object} JobService        Beer-Garden's job service.
+ */
+export default function jobExportSystemsController( $scope, $rootScope, JobService) {
+  $scope.setWindowTitle('Export');
```

Comment:
I notice that this ends up getting set as soon as you go to the scheduler page, so the scheduler page title is now just "Export".  Regardless, I don't think you want / need to do this.